mkdocs.yml

```diff
@@ -73,7 +73,8 @@ nav:
         - Galactic on Ubuntu 20: setup/ubuntu2004.md
         - Humble on Ubuntu 22: setup/ubuntu2204.md
         - Run with Docker: setup/docker.md
-        - Network Config: setup/xml-config.md
+        - Middleware Config: setup/xml-config.md
+        - Network Config: setup/network-config.md
```

Comment:
Should this be "Network Recommendations" ? OK to leave it.

Reply:
![image](https://user-images.githubusercontent.com/5374318/211658003-49310f7f-c094-4a82-9224-fa75a199d04d.png)
becomes a bit long. open to other ideas. or to merging without changing.